payments/invoice/other-features.md

```diff
@@ -19,174 +19,263 @@ sidebar:
 {% include alert.html type="warning"
                       icon="warning"
                       header="Site under development"
-                      body="This section of the Developer Portal is under construction and
-                      should not be used to integrate against Swedbank Pay's
-                      APIs yet." %}
+                      body="This section of the Developer Portal is under
+                      construction and should not be used to integrate against
+                      Swedbank Pay's APIs yet." %}
 
 ## API requests
 
 The API requests are displayed in the [invoice flow][invoice-flow].
 The options you can choose from when creating a payment with key operation
-set to Value FinancingConsumer are listed below.
+set to value `FinancingConsumer` are listed below.
 
 ### Options before posting a payment
 
 {:.table .table-striped}
-| **POST Request** | Finland ![Finish flag][fi-png]  |
-| **Operation** | FinancingConsumer |
-| **Intent** | Authorization |
-| **Currency** | EUR |
-| **InvoiceType** | PayExFinancingFI |
+|                 | Norway ![Norwegian flag][no-png] | FInland ![Finish flag][fi-png] | Sweden ![Swedish flag][se-png] |
+| :-------------- | :------------------------------- | :----------------------------- | :----------------------------- |
+| `Operation`     | `FinancingConsumer`              | `FinancingConsumer`            | `FinancingConsumer`            |
+| **Currency**    | NOK                              | EUR                            | SEK                            |
+| **InvoiceType** | `PayExFinancingNO`               | `PayExFinancingFI`             | `PayExFinancingSE`             |
 
 * An invoice payment is always two-phased based -  you create an Authorize
-    transaction, that is followed by a Capture or Cancel request.
+  transaction, that is followed by a Capture or Cancel request.
 * **Defining CallbackURL**: When implementing a scenario, it is optional
-    to set a [CallbackURL][callback-api] in the `POST` request.
-    If callbackURL is set PayEx will send a postback request to this URL when
-    the consumer has fulfilled the payment.
-    [See the Callback API description here.][callback-api]
+  to set a [CallbackURL][callback-api] in the request.
+  If callbackURL is set PayEx will send a postback request to this URL when
+  the consumer has fulfilled the payment.
+  [See the Callback API description here.][callback-api]
 
-### Authorizations
+{% include authorizations-resource.md payment-instrument="invoice" %}
 
-The `authorizations` resource will list the authorization transactions
-made on a specific payment.
+## Payment Resource
+
+{% include payment-resource.md payment-resource="invoice" %}
```

Comment:
It would be good for consistency if we used `payment-instrument` as the name of this parameter/argument/variable everywhere.

```suggestion
{% include payment-resource.md payment-instrument="invoice" %}
```